packages/ui/src/components/modals/EntityCreatorEditor/EntityCreatorContainer.tsx

```diff
@@ -182,7 +182,7 @@ class Container extends React.Component<Props, ComponentState> {
                         entityResolverVal: resolverType,
                         isPrebuilt: isPrebuilt,
                         isMultivalueVal: nextProps.entity.isMultivalue,
-                        isMultiValueDisabled,
+                        isMultiValueDisabled: isMultiValueDisabled,
```

Comment:
Prefer short-hand these, don't need to duplicate the name. `isMultiValueDisabled,`

Reply:
done